CMakeLists.txt

```diff
@@ -129,6 +129,7 @@ INCLUDE_DIRECTORIES( ${CMAKE_SOURCE_DIR}/third_party/ObjexxFCL/src/ )
 INCLUDE_DIRECTORIES( ${CMAKE_SOURCE_DIR}/third_party/SQLite/ SYSTEM )
 INCLUDE_DIRECTORIES( "${CMAKE_SOURCE_DIR}/third_party/Expat" "${CMAKE_SOURCE_DIR}/third_party/Expat/lib" SYSTEM)
 INCLUDE_DIRECTORIES( ${CMAKE_SOURCE_DIR}/third_party/CLI/ )
+INCLUDE_DIRECTORIES( ${CMAKE_SOURCE_DIR}/third_party/jsoncpp)
```

Comment:
This is the closest link I can provide

https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-96f3217ee7344807b957d62e5d778a0662307b0b-x86_64-Linux-Ubuntu-14.04-gcc-4.8-IntegrationCoverage-Debug.html